Misc/NEWS

```diff
@@ -42,6 +42,10 @@ Extension Modules
 Library
 -------
 
+- bpo-30363: Running Python with the -3 option now warns about regular
+  expression syntax that is invalid or has different semantic in Python 3
+  or will change the behavior in future Python versions.
```

Comment:
You *might* document the change in https://docs.python.org/2/whatsnew/2.7.html#porting-to-python-2-7

"or will change the behavior in future Python versions" is it possible to write code working on Python 2 and 3 that doesn't emit a warning?